src/lib.rs

```diff
@@ -69,6 +69,7 @@ pub struct Job<'a> {
     schedule: Schedule,
     run: Box<(FnMut() -> ()) + 'a>,
     last_tick: Option<DateTime<Utc>>,
+    limit_missed_runs: Option<usize>,
```

Comment:
Hi there! 

I appreciate the pull request.

I feel like this new field would be better as just a usize instead of an option. It could default to 0. Not sure it being an option is buying us much here. :)

Reply:
Updated.